Makefile

```diff
@@ -3,8 +3,11 @@ DLLINKFLAGS=-Wl,--kill-at -static-libgcc -shared
 
 all: edit-git-bash.dll
 
+# InnoSetup always uses the DLL target, and it must always be 32-bit
+# because InnoSetup is only 32-bit aware.
 edit-git-bash.dll: edit-git-bash.c
-	gcc $(CFLAGS) $(DLLINKFLAGS) -o $@ $^
+	PATH=/mingw32/bin:$$PATH \
+	i686-w64-mingw32-gcc -march=i686 $(CFLAGS) $(DLLINKFLAGS) -o $@ $^
```

Comment:
Hmmm, what if someone installs the 32-bit SDK and runs `installer/release.sh`? I think this will break because `gcc` should be called as simply `gcc` when the native environment is 32-bit.